recipe/meta.yaml

```diff
@@ -21,26 +20,46 @@ requirements:
     - {{ compiler("c") }}
     - {{ compiler("fortran") }}
     - perl
-    - make                       # [not win]
-    - cmake                      # [win]
-    - curl                       # [win]
-    - jom                        # [win]
+    - make  # [not win]
+    - cmake  # [win]
+    - curl  # [win]
+    - jom  # [win]
   host:
   run:
 
-test:
-  commands:
-    - test -f ${PREFIX}/lib/libopenblas.a                            # [unix]
-    - test -f ${PREFIX}/lib/libopenblas${SHLIB_EXT}                  # [unix]
-    - python -c "import ctypes; ctypes.cdll['${PREFIX}/lib/libopenblas${SHLIB_EXT}']"  # [unix]
-    - if not exist %PREFIX%\\Library\\lib\\openblas.lib exit 1       # [win]
-    - if not exist %PREFIX%\\Library\\bin\\openblas.dll exit 1       # [win]
+
+outputs:
+  - name: libopenblas
+    build:
+      run_exports:
+        - {{ pin_subpackage("libopenblas") }}
+    files:
+      - lib/*{{ SHLIB_EXT }}*  # [not win]
+    test:
+      commands:
+        - test -f ${PREFIX}/lib/libopenblas{{ SHLIB_EXT }}  # [not win]
+        - python -c "import ctypes; ctypes.cdll['${PREFIX}/lib/libopenblas{{ SHLIB_EXT }}']"  # [not win]
+        - if not exist %PREFIX%\\Library\\lib\\openblas.lib exit 1  # [win]
+        - if not exist %PREFIX%\\Library\\bin\\openblas.dll exit 1  # [win]
+
+  - name: openblas
+    files:
+      - include/*.h
+      - lib/*.a
+      - lib/pkgconfig/*.pc
+      - lib/cmake/openblas/*
```

Comment:
Not sure what to do on Windows. Would a `/Library/*` work for all sub-directories?
Do we need a `libopenblas` on Windows? If so it should be the `.dll`, the `.lib` or both?

Reply:
Just the `.dll`